projects/plugins/jetpack/extensions/plugins/seo/components/upsell.js

```diff
@@ -0,0 +1,57 @@
+import { useAnalytics } from '@automattic/jetpack-shared-extension-utils';
+import { Button, ExternalLink } from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import { external } from '@wordpress/icons';
+import classNames from 'classnames';
+import useUpgradeFlow from '../../../shared/use-upgrade-flow';
+import './upsell.scss';
+
+const UpsellNotice = ( { requiredPlan } ) => {
+	const [ checkoutUrl, goToCheckoutPage, isRedirecting, planData ] = useUpgradeFlow( requiredPlan );
+	const { tracks } = useAnalytics();
+
+	// Define plan name, with a fallback value.
+	const planName = planData?.product_name || __( 'paid', 'jetpack' );
+
+	const buttonText = __( 'Upgrade now', 'jetpack' );
+	const supportUrl = 'https://wordpress.com/support/seo-tools/';
+
+	const onClickHandler = event => {
+		event.preventDefault();
+		tracks.recordEvent( 'jetpack_seo_tools_upsell_click' );
```

Comment:
I'm not sure on how we test this live.

Reply:
There could be a better way but I was looking for `https://pixel.wp.com/t.gif*` requests  in the Network tab, filtering by `jetpack_seo_tools_upsell_click`

<img width="449" alt="Screenshot 2023-12-20 at 13 09 04" src="https://github.com/Automattic/jetpack/assets/1758399/6c053e05-d161-4dc2-b423-62c6f7467e2c">

Reply:
You can set `localStorage.setItem( 'debug', 'dops:analytics*' );` in your browser console, refresh the page, and you'll then get notices for each Tracks event triggered on a page you're on.

Reply:
Thanks! Yes,I was testing it locally, I thought if there's a way to see the logs outside (eg logstash)?